static/app/views/monitors/details.tsx

```diff
@@ -79,7 +79,7 @@ class MonitorDetails extends AsyncView<Props, State> {
         <Layout.Body>
           <Layout.Main fullWidth>
             {!monitor.lastCheckIn ? (
-              <MonitorOnboarding />
+              <MonitorOnboarding orgId={this.orgSlug} monitor={monitor} />
```

Comment:
```suggestion
              <MonitorOnboarding orgSlug={this.orgSlug} monitor={monitor} />
```

Reply:
I was just going to keep it consistent with all the other components in here, and will fix those in a separate PR